src/sass/partials/base/_base.scss

```diff
@@ -0,0 +1,312 @@
+
+._wrapper {
+  @include wrapper;
+}
+
+.base {
+  &--a {
+    border-bottom: 2px solid setting-get('primary color');
+    color: setting-get('primary color');
+    text-decoration: none;
+
+    &:hover {
+      border-color: setting-get('secondary color');
+      color: setting-get('secondary color');
+      transition: setting-get('fade');
+    }
+  }
+
+  // Major Third Type Scale
+
+  &--h1 {
+    font-size: 2.441em;
```

Comment:
Instead of hand-writing these values, can we please use [modular scale](https://github.com/modularscale/modularscale-sass) directly?